pom.xml

```diff
@@ -61,6 +61,18 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>${springfox.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
If we use Postgre, maybe you should not commit this to repository, just use in your local app version